usecase/parser/msg.go

```diff
@@ -152,6 +152,12 @@ func ParseBlockTxsMsgToCommands(
 func ParseMsgSend(
 	parserParams utils.CosmosParserParams,
 ) ([]command.Command, []string) {
+	// Getting possible signer address from Msh
+	var possibleSignerAddresses []string
+	if parserParams.Msg["from_address"] != nil {
+		possibleSignerAddresses = append(possibleSignerAddresses, parserParams.Msg["from_address"].(string))
+	}
```

Comment:
Do we need to cater invalid MsgSend here?

Reply:
right, updated